computation/src/main/java/com/powsybl/computation/DefaultExecutionReport.java

```diff
@@ -65,4 +63,29 @@ public void log() {
         }
     }
 
+    @Override
+    public Optional<InputStream> getStdOut(Command command, int index) {
+        return getOutputFile(command, index, ".out");
+    }
+
+    @Override
+    public Optional<InputStream> getStdErr(Command command, int index) {
+        return getOutputFile(command, index, ".err");
+    }
+
+    private Optional<InputStream> getOutputFile(Command command, int index, String extension) {
+        Objects.requireNonNull(command);
+        if (index < 0) {
+            throw new IllegalArgumentException("Invalid index: " + index);
+        }
+
+        Path path = workingDirectory.resolve(command.getId() + "_" + index + extension);
+        try {
+            return Optional.of(Files.newInputStream(path));
+        } catch (IOException e) {
+            LOGGER.warn("Unable to read {}: {}", path.toString(), e.getMessage());
```

Comment:
Sonarclound said it is not useful to add the `toString()` method as the logger already formats paths

Reply:
Fixed in the last commit